src/store/flowsStore.tsx

```diff
@@ -1,26 +1,35 @@
+// import { initialFlows } from '../stubs';
+import { useNestedStepsStore } from './nestedStepsStore';
+import useSettingsStore, { initDsl, initialSettings } from './settingsStore';
 import { StepsService } from '@kaoto/services';
 import { IFlowsWrapper, IIntegration, IStepProps, IViewProps } from '@kaoto/types';
 import { setDeepValue } from '@kaoto/utils';
 import isEqual from 'lodash.isequal';
 import { temporal } from 'zundo';
 import { create } from 'zustand';
-// import { initialFlows } from '../stubs';
-import { useNestedStepsStore } from './nestedStepsStore';
-import useSettingsStore, { initDsl, initialSettings } from './settingsStore';
 
 interface IInsertOptions {
   (integrationId: string, newStep: IStepProps, options: { mode: 'append' }): void;
-  (integrationId: string, newStep: IStepProps, options: { mode: 'insert', index: number }): void;
-  (integrationId: string, newStep: IStepProps, options: { mode: 'replace', index: number }): void;
-  (integrationId: string, newStep: IStepProps, options: { mode: 'replace', path: string[] | undefined }): void;
-  (integrationId: string, newStep: IStepProps, options: { mode: 'append' | 'insert' | 'replace'; index: number, path: string[] | undefined }): void;
+  (integrationId: string, newStep: IStepProps, options: { mode: 'insert'; index: number }): void;
```

Comment:
WebStorm did this behind me, replaced a comma with a semicolon... is this valid?